bin/pycbc_live

```diff
@@ -129,16 +176,15 @@ class LiveEventManager(object):
             else:
                 event.save(fname)
 
-    def check_singles(self, results, data_reader, psds, f_low,
-                      followup_ifos=None):
+    def check_singles(self, results, data_reader, psds, f_low):
         active = [k for k in results if results[k] != None]
         if len(active) == 1:
             ifo = active[0]
             single = sngl_estimator[ifo].check(results[ifo], data_reader[ifo])
 
             if single is not None:
                 fud = self.compute_followup_data([ifo], single, data_reader,
-                                                 bank, followup_ifos)
+                                                 bank, [])
```

Comment:
Do I understand right that this disables multidetector skymaps for single events? Did the patch break functions in a way that requires this?

Reply:
This situation doesn't occur in the current code. I.E. there are *no* followup detectors in the old sense. They are currently all used for significance if there. So the only case is if they are on or not. If they are on they are used for skymap and for significance. Currently, a single only occurs if the other detectors are offline, so there can't be a multidetector skymap.